agent/exploits/cve_2023_1389.py

```diff
@@ -56,7 +56,7 @@ def _generate_vulnerability_object(
     ) -> definitions.Vulnerability:
         entry = kb.Entry(
             title=VULNERABILITY_TITLE,
-            risk_rating="CRITICAL",
+            risk_rating="POTENTIALLY",
```

Comment:
I think we should improve the detection instead of changing risk rating

Reply:
We need to have known vulnerable instances to do so.

Reply:
we need a call back or we need a vulnerable instance